release/nightly_tests/setup_chaos.py

```diff
@@ -1,24 +1,29 @@
 import argparse
 import ray
 
-from ray._private.test_utils import get_and_run_node_killer
+from ray._private.test_utils import (
+    get_and_run_resource_killer,
+    NodeKillerActor,
+    WorkerKillerActor,
+)
 
 
 def parse_script_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--node-kill-interval", type=int, default=60)
-    parser.add_argument("--max-nodes-to-kill", type=int, default=2)
+    parser.add_argument("--kill-workers", action="store_true", default=False)
+    parser.add_argument("--kill-interval", type=int, default=60)
+    parser.add_argument("--max-to-kill", type=int, default=2)
```

Comment:
we will also need to modify the parameters used in the existing release test definitions in `release_test.yaml` which use `setup_chaos.py`.